src/containers/Accounts/index.js

```diff
@@ -26,23 +27,13 @@ ERROR_MESSAGES.default = {
 
 const getErrorMessage = error => ERROR_MESSAGES[error] || ERROR_MESSAGES.default;
 
-class Accounts extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      currencySelected: 'XRP',
-    };
-  }
-
-  static getDerivedStateFromProps(nextProps, prevState) {
-    const { match } = nextProps;
-    return {
-      accountId: match.params.id,
-      prevId: prevState && prevState.accountId,
-    };
-  }
+const Accounts = props => {
+  const { id } = useParams();
```

Comment:
```suggestion
  const { id: accountId } = useParams();
```
It's a little more descriptive of a variable name